web3/eth/eth.py

```diff
@@ -619,15 +619,15 @@ def modify_transaction(
 
     # eth_newFilter, eth_newBlockFilter, eth_newPendingTransactionFilter
 
-    filter: Method[Callable[[Optional[Union[str, FilterParams, HexStr]]], Filter]] = (
-        Method(
-            method_choice_depends_on_args=select_filter_method(
-                if_new_block_filter=RPC.eth_newBlockFilter,
-                if_new_pending_transaction_filter=RPC.eth_newPendingTransactionFilter,
-                if_new_filter=RPC.eth_newFilter,
-            ),
-            mungers=[BaseEth.filter_munger],
-        )
+    filter: Method[
```

Comment:
My `black` has been reformatting some things lately locally. Anyone else having this issue? Not seeing why this needed to change.

Reply:
nm fixed... I must have conflicting versions